### ISSUE TITLE
'storageAccessFrameworkFromAPI' attibute missing - AIRSDK 51.0.0.2 

### ISSUE THREAD

**Issue by wmtj** [airsdk/Adobe-Runtime-Support#3092]
D:\work\xxx\bin-debug\app.xml(315): error 103: application.android.storageAccessFrameworkFromAPI is an unexpected element/attribute

**Comment by ajwfrost** [comment]
Yes we noticed that earlier :-(   the validation class for namespace 51.0 was deriving from 50.1 rather than 50.2 so there are two definitions that ADT will throw an error for.
We're currently building an updated version of the 51.0 release branch so this should be possible when that comes out, trying to make it a quick turnaround!

thanks
